### DISCUSSION TITLE
Possible issue in number of STAICoin farmed counter

### DISCUSSION THREAD

**Discussion by LeroyINC** [guydavis/machinaris#486] [General]
Juts an FYI.. looks like the display counter for STAIcoin for the total farmed is always 0
I am sure this is juts cosmetic as the counter for my total balance is correct.

**Comment by guydavis** [comment]
Hi, please try trouble-shooting your install, including a possible status database reset.

I am not seeing missing STAI balance on either the Summary or Wallet page.

Please let me know how it goes. Thanks!

**Comment by LeroyINC** [comment]
Wallet is correct..

summary page is showing balance correct.. but total farmed is always 0  ... i will see what i can debug

**Reply by LeroyINC** [reply]
I still have this issue where the reported amount for the amount farmed for STAI coin is 0.

this is strictly cosmetic... did some digging and found this in the debug log for stai coin

2022-01-05T12:39:50.926 wallet stai.rpc.rpc_server        : WARNING  Error while handling message: Traceback (most recent call last):
  File "/staicoin-blockchain/stai/rpc/rpc_server.py", line 83, in inner
    res_object = await f(request_data)
  File "/staicoin-blockchain/stai/rpc/wallet_rpc_api.py", line 1153, in get_farmed_amount
    assert amount == pool_reward_amount + farmer_reward_amount + fee_amount
AssertionError


this looks to be an error in the stai coin code if i am not mistaken
